src/Domain/TemplateTestCampaign.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+namespace WMDE\Fundraising\Frontend\Domain;
+
+/**
+ * @licence GNU GPL v2+
+ * @author Leszek Manicki <leszek.manicki@wikimedia.de>
+ * @author Kai Nissen < kai.nissen@wikimedia.de >
+ */
+class TemplateTestCampaign {
```

Comment:
Things in the domain cannot know about templates, which are a presentation concern